src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -2007,6 +2007,23 @@ public function only($attributes)
         return $results;
     }
 
+    /**
+     * Get all attributes except the given ones.
+     *
+     * @param  array|mixed  $attributes
+     * @return array
+     */
+    public function except($attributes)
+    {
+        $results = $this->attributesToArray();
+
+        foreach (is_array($attributes) ? $attributes : func_get_args() as $attribute) {
+            unset($results[$attribute]);
+        }
+
+        return $results;
```

Comment:
We have [`Arr::except`](https://laravel.com/docs/12.x/helpers#method-array-except) which could be used to improve this.

```suggestion
        return Arr::except($this->attributesToArray(), is_array($attributes) ? $attributes : func_get_args());
```

Reply:
Fixed. Thanks!